src/web/services/course.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { MessageOutput } from '../types/api-output';
+import { HttpRequestService } from './http-request.service';
+
+/**
+ * Handles sessions related logic provision.
+ */
+@Injectable({
+  providedIn: 'root',
+})
+export class CourseService {
```

Comment:
You can use `ng g s CourseService` so that Angular CLI will generate corresponding test file also. Instead of copy here.